astronomer/providers/apache/hive/example_dags/example_hive.py

```diff
@@ -277,6 +279,16 @@ def get_cluster_details(task_instance: Any) -> None:
     )
 
 
+def check_dag_status(**kwargs: Any) -> None:
```

Comment:
As this is repeated in multiple DAGs, should we extract it to avoid duplication?

Reply:
I had this thought but have following next thoughts:
1. These are example DAGs and need to be self sufficient
2. We may not want to keep this as a util and release it as part of our providers

Owing to the above reasons, I guess the duplication should be okay.